explainer-server/public/javascripts/explain-editor-plain-js.js

```diff
@@ -43,13 +43,26 @@ function updateCheckboxState() {
  * Tag Search
  */
 
-function processCapiSearchResponse(response){
+
+function processCapiSearchResponseTags(divIdentifier,response,userInterfaceTagDescriptionKey){
+    response.results.forEach(function(tag){
```

Comment:
You asked me the same question last week :)
Answer no yet. Eventually yes, but not now. I need to study the conversion of JS values to Scala values first and that's a tricky one. It's on my todo list though.